src/main/groovy/com/github/jengelman/gradle/plugins/shadow/tasks/ShadowJar.java

```diff
@@ -31,7 +31,7 @@
 
     private List<Transformer> transformers;
     private List<Relocator> relocators;
-    private transient List<Configuration> configurations;
+    private List<FileCollection> configurations;
```

Comment:
I'm not sure whether the removal of `transient` can affect correctness of outputs; I was surprised that I didn't notice any differences in output between my first and second builds even though the second build was not UP-TO-DATE.

However, removing `transient` does seem to allow the task to be up-to-date in the second build.

cc @eskatos since it seems to have been added here: https://github.com/johnrengelman/shadow/commit/c9e30f10aecd69600213a2853296d9e6175a715d#diff-d5abeaae583093a8a92ef53ffe7f5a21cf185cf0d5e3a76edd236fa26dbf7e9fR34

Reply:
Good call! Looks good to me

Reply:
Thanks!